<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QLineEdit" name="calculator_display">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plusButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minusButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiplyButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="divideButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="oneButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="twoButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="threeButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fourButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fiveButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sixButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sevenButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eightButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nineButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zeroButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="equalButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decimalButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftParenthesis">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>16</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>(</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightParenthesis">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>16</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
